README.md

```diff
@@ -46,18 +76,38 @@ cmake ../
 make -j
 ```
 
-After that, in the `build` folder will be the compiled library code as well
-as generated samples.
+On Windows there are a couple of options worth looking into. CMake has a GUI
+which allows you to select projects and how to build them, if you use the GUI,
+you can directly select "MinGW" as the generator to ensure that the build
+takes place properly. Or you can make use of CLion which will abstract the
+build process away. Another option (which I recommend for long term use), is
+to learn about WSL (Windows Subsystem for Linux). Linux is the OS of choice
+for software developers and includes a lot of great tools for development. WSL
+is a happy medium between running Windows and installing a virtual machine.
 
-For building with linting enabled (recommended). First install cpplint, then
-run the installation with the modification below.
+After running those commands, you should have a fully build EVT-core including
+samples.
 
-```bash
-mkdir build/
-cd build/
-cmake -DEVT_LINT=ON ../
-make -j
-```
+For a more detailed build process using CLion and on Windows,
+visit the EVT wiki.
```

Comment:
I'd suggest linking directly to the wiki here.

Reply:
Good point, adding it